source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -242,30 +219,18 @@ void BioXASAppController::goToEnergyCalibrationView(AMScan *toView)
 	}
 }
 
-void BioXASAppController::onCurrentScanActionStartedImplementation(AMScanAction *action)
-{
-	Q_UNUSED(action)
-
-	// Save current configuration to the database.
-
-	if (bioxasUserConfiguration_)
-		bioxasUserConfiguration_->storeToDb(AMDatabase::database("user"));
-}
-
 void BioXASAppController::onCurrentScanActionFinishedImplementation(AMScanAction *action)
 {
-	// Save current user configuration to the database.
-
-	if (bioxasUserConfiguration_)
-		bioxasUserConfiguration_->storeToDb(AMDatabase::database("user"));
-
 	// If the scan was an energy calibration scan, set the calibration view's scan and make it the current pane.
 
 	if (action && action->controller() && action->controller()->scan()) {
 		BioXASXASScanConfiguration *xasScanConfiguration = qobject_cast<BioXASXASScanConfiguration*>(action->controller()->scan()->scanConfiguration());
 		if (xasScanConfiguration && xasScanConfiguration->name() == "Energy Calibration XAS Scan")
 			goToEnergyCalibrationView(action->controller()->scan());
 	}
+
```

Comment:
Just checking. Does execution order matter here? Previously, the store to db happened before any of the view changes.